Sources/SwiftDocC/Model/TaskGroup.swift

```diff
@@ -200,18 +200,15 @@ public struct TaskGroup {
             return nil
         }
         
-        let startIndex: Int
-        if originalContent.first is Paragraph {
-            startIndex = 1
-        } else {
-            startIndex = 0
-        }
+        var discussionChildren = originalContent
+            .prefix(while: { !($0 is UnorderedList) })
```

Comment:
Is there any way to write these closures using positive logic? So instead of not-UnorderedList and not-BlockDirective can you write `is Paragraph`? What types can we have here? If a child is not an UnorderedList and not a BlockDirective does it have to be a paragraph?

Reply:
Sounds good - the negative logic makes sense then. thanks.